src/frontend/tsx/Challenges.tsx

```diff
@@ -77,16 +79,45 @@ export function Challenges () {
     await pokeAction(urbit, challenge(who, side, description), onError, onSuccess)
   }
 
+  const findFriends = async () => {
+    const friendsScry: Object = await urbit.scry({ app: 'chess', path: '/friends' })
+    const stringified: string = JSON.stringify(friendsScry, null, 2)
+    const parsed = JSON.parse(stringified) as { friends: Array<Ship> }
+    setFriends(parsed.friends)
+  }
```

Comment:
This function should go in `ts/helpers/urbitChess` - that's the default place for pokes/scries